GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java

```diff
@@ -599,6 +599,9 @@ private void processArrow(Arrow arrow) {
             case LeaveRoundaboutEasyRight:
                 garminHud.SetDirection(eOutAngle.EasyRight, eOutType.RightRoundabout, eOutAngle.EasyRight);
                 break;
+            case LeaveRoundaboutAsDirection:
+                garminHud.SetDirection(eOutAngle.Down, eOutType.RightRoundabout, eOutAngle.Down);
```

Comment:
What is LeaveRoundaboutAsDirection arrow like?
Could you show some image?

Reply:
I remove LeaveRoundaboutAsUp, replace with LeaveRoundaboutAsDirection, keep same rule as other arrow.

Reply:
LeaveRoundaboutAsDirection is this symbol:
![image](https://user-images.githubusercontent.com/33813894/52915698-80b1bc80-32d7-11e9-907b-1f4dbccbeae8.png)
(Means: Use Roundabout to make a turn)

Reply:
Got it, I miss understand it.